config/initializers/nationalities.rb

```diff
@@ -228,3 +228,44 @@
 
 NATIONALITIES_BY_NAME = NATIONALITIES.map(&:reverse).to_h
 NATIONALITY_DEMONYMS = NATIONALITIES.map(&:second)
+
+# Last updated on 21/12/2020. To update run the following code in irb and update the EU_COUNTRY_CODES array with the result.
+# require 'net/http' # require 'json'
+# response = Net::HTTP.get('restcountries.eu', '/rest/v2/regionalbloc/eu?fields=alpha2Code;name')
+# JSON.parse(response).inject([]) { |response, country| response << [ country['alpha2Code'], country['name'] ] }
+
+EU_COUNTRY_CODES = [
+  'AX',
+  'AT',
+  'BE',
+  'BG',
+  'HR',
+  'CY',
+  'CZ',
+  'DK',
+  'EE',
+  'FO',
+  'FI',
+  'FR',
+  'GF',
+  'DE',
+  'GI',
+  'GR',
+  'HU',
+  'IE',
+  'IM',
+  'IT',
+  'LV',
+  'LT',
+  'LU',
+  'MT',
+  'NL',
+  'PL',
+  'PT',
+  'RO',
+  'SK',
+  'SI',
+  'ES',
+  'SE',
+  'GB'
```

Comment:
With 11 days to go and the GB precedence rule it may be worth removing this last line :-/ 

There's also a gov.uk page at https://www.gov.uk/eu-eea but doesn't contain codes.

Reply:
I did think that, however as this nationality would never be taken into account anyway I decided not to edit the returned service data.

Reply:
We could work with names if we can get that JSON, but I don't think they provide that anywhere 🤷🏼‍♀️

Reply:
Yes, like you say it doesn't make much difference.